ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/deduplication/CommandDeduplicationBase.scala

```diff
@@ -315,4 +301,72 @@ private[testtool] abstract class CommandDeduplicationBase(
       )
     }
   })
+
+  def requestHasOkCompletion(
+      ledger: ParticipantTestContext
+  )(request: SubmitRequest, party: Party)(implicit ec: ExecutionContext): Future[Completion] = {
+    requestHasCompletionWithStatusCode(ledger)(request, party, Code.OK)
+  }
+
+  def requestHasCompletionWithStatusCode(
+      ledger: ParticipantTestContext
+  )(request: SubmitRequest, party: Party, code: Code)(implicit
+      ec: ExecutionContext
+  ): Future[Completion] = {
+    submitRequestAndFindCompletion(ledger)(request, party).map(completion => {
+      assert(
+        completion.getStatus.code == code.value(),
+        s"Expecting completion with status code $code but completion has status ${completion.status}",
+      )
+      completion
+    })
+  }
```

Comment:
Nit: unneeded braces

Reply:
Doing another cleanup in a PR that I'm about to open so I'll change all these braces there.